docker-entrypoint.sh

```diff
@@ -0,0 +1,8 @@
+cp $KLAYTN_CONTRACT_DEPLOYMENT_DIR/networks-DexFactory.json /app/subgraphs/exchange/networks.json
```

Comment:
What do you think of adding this line?

```suggestion
#!/bin/bash

cp $KLAYTN_CONTRACT_DEPLOYMENT_DIR/networks-DexFactory.json /app/subgraphs/exchange/networks.json
```

Reply:
I have /bin/bash in entrypoint

Reply:
Added in script also